### ISSUE TITLE
Metric computation on a test set with a Pretrained model

### ISSUE THREAD

**Issue by rohitpaturi** [pyannote/pyannote-audio#368]
Is there a way to compute Fscore for a given test set and a pre-trained model for overlap detection? You have a visualization tool mentioned here https://github.com/pyannote/pyannote-audio/tree/develop/tutorials/pretrained/model but it would be great to compute the metrics for a given test set. Thanks!

**Comment by hbredin** [comment]
[`pyannote.metrics`](http://pyannote.github.io/pyannote-metrics/reference.html#pyannote.metrics.detection.DetectionPrecisionRecallFMeasure) provides a bunch detection metrics that shoud help you with that.

Also, [this piece of code](https://github.com/pyannote/pyannote-audio/blob/f73eaa436cf0147753ea3f67002d76cbd0b98d7e/pyannote/audio/pipeline/overlap_detection.py#L152-L173) shows how to compute overlapping regions starting from a reference annotation and [this one ](https://github.com/pyannote/pyannote-audio/blob/f73eaa436cf0147753ea3f67002d76cbd0b98d7e/pyannote/audio/pipeline/overlap_detection.py#L219-L232) is an example of how to use the aforementionned metrics for evaluating overlapped speech detection.

**Comment by hbredin** [comment]
I took the liberty to create [a new issue](#374) for this new question.
